docs/src/developing_zed__building_zed_windows.md

```diff
@@ -0,0 +1,76 @@
+# Building Zed for Windows
+
+## Repository
+
+After cloning the repository, ensure all git submodules are initialized:
+
+```shell
+git submodule update --init --recursive
+```
+
+## Dependencies
+
+- Install [Rust](https://www.rust-lang.org/tools/install). If it's already installed, make sure it's up-to-date:
+
+  ```bash
```

Comment:
Is bash required? If not, should we be using PowerShell or some such here. If so, should we make a note that this needs to be run with WSL or something like that? 

Or are these already valid in PowerShell?

Reply:
as those are just for telling github how to syntax highlight I think it's fine, I think this is just a remnant of: https://github.com/zed-industries/zed/blob/main/docs/src/developing_zed__building_zed_linux.md?plain=1 edited for windows. Also rustup should work fine on windows powershell or the like according to its docs so no need for wsl.

Reply:
Since `bash` might make the mistake of requiring WSL, we might use `pwsh` or simply follow the part that uses `shell`.
Or, since the syntax highlighting is not that important, we may simply not specify anything.

I've tried this, and both give slightly different results. Which one looks better?

pwsh:
<img width="169" alt="image" src="https://github.com/zed-industries/zed/assets/6465609/5c949978-5711-4e7a-bd8e-a0a4dd61fcca">

shell:
<img width="151" alt="image" src="https://github.com/zed-industries/zed/assets/6465609/9cfb4b5e-7231-42da-b420-2dc489f1250d">

Reply:
I know about the markdown syntax highlighting hint. I meant to ask 'are the contents of these bash sections correct'.

Reply:
For example, would an annotation like this look good?

> [!NOTE]
> The following commands may be executed in any shell.

Reply:
Yes!